OpenRA.Game/Graphics/UISpriteRenderable.cs

```diff
@@ -40,6 +42,14 @@ public UISpriteRenderable(Sprite sprite, WPos effectiveWorldPos, int2 screenPos,
 				this.palette = null;
 		}
 
+		public UISpriteRenderable(Sprite sprite, WPos effectiveWorldPos, int2 screenPos, int zOffset, PaletteReference palette)
+			: this(sprite, effectiveWorldPos, screenPos, zOffset, palette, 1f, 1f, 0f) { }
+		public UISpriteRenderable(Sprite sprite, WPos effectiveWorldPos, int2 screenPos, int zOffset, PaletteReference palette, float scale)
+			: this(sprite, effectiveWorldPos, screenPos, zOffset, palette, scale, 1f, 0f) { }
+
+		public UISpriteRenderable(Sprite sprite, WPos effectiveWorldPos, int2 screenPos, int zOffset, PaletteReference palette, float scale, float alpha)
```

Comment:
Yes however you will note this is the original ctor before rotation was implemented, as it has both scale and alpha parameters, while setting rotation to 0f. Note also that float and alpha are no longer the last two parameters, so a default rotation of 0f now needs to be passed when using the original ctor. I have reworked the calls to UISpriteRenderable to take the rotation parameter, which is why this ctor is not being referenced, however I think it is worth keeping this for any cases where rotation is not needed, since it is the only way to call UISpriteRenderable without passing in a rotation.